core/src/test/scala/org/apache/spark/sql/delta/DeltaTimeTravelSuite.scala

```diff
@@ -46,6 +48,11 @@ class DeltaTimeTravelSuite extends QueryTest
     duration.toMillis
   }
 
+  protected override def sparkConf = {
```

Comment:
The change in this file is not needed since the test suite extends `DeltaSQLCommandTest`.